src/rust-crypto/rust-crypto.ts

```diff
@@ -1760,6 +1761,49 @@ export class RustCrypto extends TypedEventEmitter<RustCryptoEvents, RustCryptoEv
     public async getOwnIdentity(): Promise<RustSdkCryptoJs.OwnUserIdentity | undefined> {
         return await this.olmMachine.getIdentity(new RustSdkCryptoJs.UserId(this.userId));
     }
+
+    public async encryptToDeviceMessages(
```

Comment:
We normally try and give some sort of signposting doc-comment:

```suggestion
    /**
     * Implementation of {@link CryptoApi#encryptToDeviceMessages}.
     */
    public async encryptToDeviceMessages(
```

Also, please can you stick it alongside the other CryptoApi method implementations (note there are a few box comments scattered throughout the `RustCrypto` class which attempt to divide it into sections).